examples/multipart-cloud-init/data.tf

```diff
@@ -2,7 +2,7 @@ data "template_file" "init" {
   count = length(var.hostnames)
   template = file("cloud-init.yml.tpl")
 
-  vars {
+  vars = {
     fqdn = var.hostnames[count.index]
     ubuntu = "artful"
```

Comment:
bionic?

Reply:
Thanks, I changed for 18.04 in `variables.tf` so you're right